browser-extension/mv2/src/background/background.js

```diff
@@ -1,3 +1,75 @@
+// import { startEventBatchWriter } from "./EventWriter";
+
+/* can be moved to a different file --- EventWriter.js */
+
+let eventsBatch = [];
```

Comment:
Lets name this `events` or `analyticEvents`. We shouldn't use the word `batch` unless batches are created. Its confusing